### ISSUE TITLE
Fix CCS ok/x display on auto-approve page

### ISSUE THREAD

**Issue by matschaffer** [Safecast/safecastapi#812]
![Notification_Center](https://user-images.githubusercontent.com/690/118387746-24c37300-b65b-11eb-87e4-b9944b6fed46.png)

There should be icons visible on this tab showing ok/x for auto approval status. Looks like we lost them during the bootstrap CCS upgrade.

Would be good to get those back.

**Comment by nokton** [comment]
Or replace with simple “fail/pass” text so we can’t loose any more icons 🙏

**Comment by ZacharyCChang0828** [comment]
I thought it makes sense for me to work on this since I used to be working mainly on auto approval stuffs. 

I was able to bring the icons back and I also added plain texts which I guess kind of defeat the purpose of having icons. If we want to stick to using plain text I think we will need to make some modification to the process log tab too.

![image](https://user-images.githubusercontent.com/35931293/120500360-5ece9c00-c3f3-11eb-83fe-d1ac7c361da7.png)
